Bottom: 2320800b501a6466c5d367bf27d76de089454cc9
Top:    a9e6607be07dff65b7d95485c0884283f2ce9536
Author: Igor Stoppa <igor.stoppa@huawei.com>
Date:   2018-05-03 15:28:32 +0400

Refresh of preliminary-self-test-for

---

diff --git a/mm/Kconfig b/mm/Kconfig
index d7ef40eaa4e8..d5004d82a1d6 100644
--- a/mm/Kconfig
+++ b/mm/Kconfig
@@ -752,9 +752,3 @@ config GUP_BENCHMARK
 	  performance of get_user_pages_fast().
 
 	  See tools/testing/selftests/vm/gup_benchmark.c
-
-config PROTECTABLE_MEMORY
-    bool
-    depends on MMU
-    depends on ARCH_HAS_SET_MEMORY
-    default y
